src/Nether.Analytics.DataLake/DataLakeStoreOutputManager.cs

```diff
@@ -45,27 +50,32 @@ public DataLakeStoreOutputManager(IMessageFormatter serializer, IFilePathAlgorit
         }
 
 
-        public async Task OutputMessageAsync(string pipelineName, int idx, Message msg)
+        public Task OutputMessageAsync(string partitionId, string pipelineName, int idx, Message msg)
```

Comment:
Why is there the new partitionId parameter? The Message type that is passed in has a PartitionId property.